preliz/distributions/continuous.py

```diff
@@ -16,6 +16,26 @@
 eps = np.finfo(float).eps
 
 
+class TauSigma:
```

Comment:
No need to define a class, just define the two functions and call them from within each distribution you need to.